### ISSUE TITLE
feature request: some way to randomize data

### ISSUE THREAD

**Issue by jftiv** [TylerReid/bloop#9]
experimenting with this on the side, similar to how you have stuff like {{$guid}} in postman, would be nice to have a regex driven way to generate stuff for request bodies. 

was messing with something like $string.12 or $number.8 (type.length)

**Comment by TylerReid** [comment]
Yeah, I could see that being useful. One way you can do that now is a variable configured to run a PowerShell or bash script, which is the general escape hatch to do whatever crazy thing you want. But it might be nicer to have some built in functions.
